src/cc/api/BPF.h

```diff
@@ -137,6 +137,10 @@ class BPF {
     return BPFPercpuHashTable<KeyType, ValueType>({});
   }
 
+  void* get_bsymcache(void) {
```

Comment:
How does this work? I imagine user would get this (bsymcache) to add module. But how is it initialized?

If you could also add a C++ example that would be great:))!

Reply:
test_bpf_table.cc has the required example as a test-case. Also, it is modeled similar to BPF python object. As suggested in another comment, I moved the bsymcache_ at the BPF level.